### PR TITLE
fix/@charterAfrica not starting

### LINKED ISSUE #439: @charterAfrica failing to start
**Describe the bug**
PR #430  updated the `countries` JSON data to include different locales. openAfrica integration does not take into account locales when formatting the countries list. This causes running the app to fail.

**To Reproduce**
Running current main branch.

**Expected behavior**
The main branch should start correctly and country labels should be displayed as per the different locales.

### PR DESCRIPTION
**Pull request by kelvinkipruto**
## Description
This PR fixes a bug introduced by #430 

Fixes #439

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: apps/charterafrica/src/components/Datasets/useDatasets.js [hunk 1]
```diff
@@ -4,11 +4,14 @@ import queryString from "@/charterafrica/utils/datasets/queryString";
 
 const fetcher = (url) => fetch(url).then((res) => res.json());
 
-function useDatasets(query, pathname) {
+function useDatasets(query, pathname, locale) {
```
Comment (kilemensi) [inline apps/charterafrica/src/components/Datasets/useDatasets.js:7]: `locale` should be part of `query` & should be handled by `queryString`. See articles' `queryString` implementation: https://github.com/CodeForAfrica/ui/blob/045bd61ba810c1d34ee3b7e11fc689da8635c00c/apps/charterafrica/src/utils/articles/queryString.js#L12

File: apps/charterafrica/src/lib/openAfrica.js [hunk 2]
```diff
@@ -126,14 +127,14 @@ export default async function fetchDatasets(
...
-    const formattedData = formatResponse(response, pathname);
+    const formattedData = formatResponse(response, pathname, locale);
     return formattedData;
   } catch (error) {
     return error;
   }
 }
 
-export async function fetchDataset(id, path) {
+export async function fetchDataset(id, path, locale = "en") {
```
Comment (kilemensi) [inline apps/charterafrica/src/lib/openAfrica.js:137]:
```suggestion
export async function fetchDataset(id, pathname, query) {
```

1. `locale` should be part of `query`.
2. use `pathname` consistently ... mixing `path` and `pathname` will lead to confusion down the line.

**Review by kilemensi — COMMENTED** [review]
👍🏽 ... Kudos for quickly spotting this & fixing it. We need to up our testing game.

**Review by koechkevin — APPROVED** [review]

File: apps/charterafrica/src/pages/api/v1/resources/datasets.page.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ async function datasets(req, res) {
       q = "",
       page = 1,
       path: pathname,
```
Comment (kilemensi) [inline apps/charterafrica/src/pages/api/v1/resources/datasets.page.js:12]: Why are we passing this as `path`?

File: apps/charterafrica/src/lib/openAfrica.js [hunk 4]
```diff
@@ -73,23 +73,23 @@ function formatDatasets(datasets, path) {
...
 
   const datasets = formatDatasets(results, pathname);
   const sortStrings = (a, b) => a.localeCompare(b);
   const tagsList = Object.keys(tags || {}).sort(sortStrings);
-  const countries = countriesByContinent("Africa").map(({ name, value }) => ({
-    value: value.toLowerCase(),
-    label: name,
+  const countries = countriesByContinent("Africa").map(({ label, value }) => ({
+    value,
+    label: label[locale].toLowerCase(),
```
Comment (kilemensi) [inline apps/charterafrica/src/lib/openAfrica.js:92]: Is there a need for `label` to be lower case?

**Review by kilemensi — APPROVED** [review]
🚀

**Pull request merged**